lib/src/web_socket/browser/sockjs.dart

```diff
@@ -79,12 +81,24 @@ class SockJSWebSocket extends CommonWebSocket implements WebSocket {
     // an error if the socket moves straight to the closed state.
     final connected = new Completer<Null>();
     // ignore: unawaited_futures
-    client.onOpen.first.then((_) => connected.complete());
+    client.onOpen.first.then((_) {
```

Comment:
This didn't actually change in this PR, it's always been this way, but because there will only ever be one event. The sockJS library just happens to be designed such that all events are dispatched via streams, even if it's a one-time thing